recipes/tip/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "tip" %}
+{% set version = "v1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4ba6f6a9e652fa48a99534997aaaa0921a94af0ef2d65ce909b6825c06a2b259
+
+build:
+  skip: true  # [win and vc<14 or osx]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('tip', max_pin='x.x') }}
+  script_env: 
+    - TIP_VERSION={{ version }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make ==4.3  # [unix]
+    - ninja ==1.10.2
+    - cmake ==3.21.3
+    - pkgconfig ==1.5.5 
+    - libpcap ==1.10.1
+    - arrow-cpp ==6.0.2
+    - spdlog ==1.8.5
+    - yaml-cpp ==0.6.3
+    - gmock ==1.10.0
+    - gtest ==1.10.0
+    - libtins ==4.2
```

```suggestion
  build:
    - {{ compiler('cxx') }}
    - make ==4.3  # [unix]
    - ninja ==1.10.2
    - cmake ==3.21.3
    - pkgconfig ==1.5.5 
```
No libraries should be listed in the build section. Build is for tools only.